frontend/src/app/main/data/comments.cljs

```diff
@@ -338,3 +409,43 @@
 
       (= :yours mode)
       (filter #(contains? (:participants %) (:id profile))))))
+
+(defn update-comment-thread-frame
+    ([thread ]
+   (update-comment-thread-frame thread uuid/zero))
+
+   ([thread frame-id]
+    (us/assert ::comment-thread thread)
+    (ptk/reify ::update-comment-thread-frame
+      ptk/UpdateEvent
+      (update [_ state]
+        (let [thread-id (:id thread)]
+          (assoc-in state [:comment-threads thread-id :frame-id] frame-id)))
+
+      ptk/WatchEvent
+      (watch [_ _ _]
+        (let [thread-id (:id thread)]
+          (->> (rp/mutation :update-comment-thread-frame {:id thread-id  :frame-id frame-id})
+               (rx/catch #(rx/throw {:type :update-comment-thread-frame}))
```

Comment:
The `:type` should be part of a general keyword part of some category. Something like `:comment-error` and the `:code` can be something that describes the concrete cause.